pkg/chain/local/local.go

```diff
@@ -169,7 +169,7 @@ func (c *localChain) SubmitRelayEntry(newEntry []byte) *async.EventEntrySubmitte
 
 	currentBlock, err := c.blockCounter.CurrentBlock()
 	if err != nil {
-		relayEntryPromise.Fail(fmt.Errorf("cannot read current block"))
+		_ = relayEntryPromise.Fail(fmt.Errorf("cannot read current block"))
```

Comment:
Let's log errors from failed promises in this file. It's a lot of boilerplate code but it's better than ignoring those errors.